src/utils/logger.ts

```diff
@@ -7,6 +7,14 @@ export default class Logger {
     console.log(message)
   }
 
+  public logSuccess(object_reference: string, event_reference: string, timeReceived: string, timeSent: string): void {
+    this.log('Message successfully sent to SNS: ' +
+    '- object reference: ' + object_reference +
+    '- event reference: ' + event_reference +
+    '- time message received: ' + timeReceived +
+    '- time message sent: ' + timeSent)
+  }
```

Comment:
Does something like this achieve the same result? 
```suggestion
    this.log(`Message successfully sent to SNS: 
    - object reference:  ${object_reference}
    - event reference:  ${event_reference} 
    - time message received: ${timeReceived} 
    - time message sent: ${timeSent}`)
  }
```